node/resolvers/directives/checkUserAccess.ts

```diff
@@ -1,7 +1,65 @@
-import { SchemaDirectiveVisitor } from 'graphql-tools'
+import { AuthenticationError, ForbiddenError } from '@vtex/api'
 import type { GraphQLField } from 'graphql'
 import { defaultFieldResolver } from 'graphql'
-import { AuthenticationError, ForbiddenError } from '@vtex/api'
+import { SchemaDirectiveVisitor } from 'graphql-tools'
+
+export async function checkUserOrAdminTokenAccess(
+  ctx: Context,
+  token?: string,
+  operation?: string
+) {
+  const {
+    vtex: { storeUserAuthToken, logger },
+    clients: { identity, vtexId },
+  } = ctx
+
+  if (!token && !storeUserAuthToken) {
+    logger.warn({
+      message: `CheckUserAccess: No admin or store token was provided for ${operation}`,
+      operation,
+    })
+    throw new AuthenticationError('No admin or store token was provided')
+  }
+
+  if (token) {
+    try {
+      await identity.validateToken({ token })
+    } catch (err) {
+      logger.warn({
+        error: err,
+        message: `CheckUserAccess: Invalid admin token for ${operation}`,
+        operation,
+        token,
```

Comment:
This PR is just missing the changes you made on that other similar for storefront-permissions, where you removed the `token` field from the logs and also removed `operation` from the log messages.